java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/OutgoingAsync.java

```diff
@@ -218,7 +218,7 @@ public final boolean completed(InputStream is) {
 
         // _is can already be initialized if the invocation is retried
         if (_is == null) {
-            _is = new InputStream(_instance, Protocol.currentProtocolEncoding);
```

Comment:
The new ctor for these empty buffer InputStream now automatically sets the encoding to Protocol.currentProtocolEncoding.